cmake/Config.cmake

```diff
@@ -19,16 +19,36 @@ if (WIN32)
             "${CMAKE_CURRENT_SOURCE_DIR}/cs/test/compat/bin/retail")
 endif()
 
-find_package (PythonLibs 2.7)
+# find python interpreter, library and boost python library
+# to specify a different version, invoke cmake with:
+# -DPYTHON_EXECUTABLE=/path/to/python
+# -DPYTHON_LIBRARY=/path/to/libpython.so
+# -DBoost_PYTHON_LIBRARY=/path/to/libboost-python.so
+# and optionally with:
+# -DPython_ADDITIONAL_VERSIONS=Major.Minor
+# if your python version is not implicitly supported by cmake
 find_package (PythonInterp 2.7)
+find_package (PythonLibs 2.7)
+
+# to modify boost python auto-detection for your system, modify
+# "python" below to "pythonlibname" in libboost<pythonlibname>.so
```

Comment:
It won't work properly with library named anything other than `python` because the makefiles use `Boost_PYTHON_LIBRARY/FOUND` variables. I'd just remove this comment. The comment above describes how to use different library/version of Python.